src/main/java/net/mcreator/element/ModElementTypeLoader.java

```diff
@@ -52,13 +52,11 @@ public static void loadModElements() {
 				new ModElementType<>("enchantment", 'm', BaseType.ENCHANTMENT, RecipeType.NONE, EnchantmentGUI::new,
 						Enchantment.class));
 		ModElementType.FLUID = register(
-				new ModElementType<>("fluid", 'u', BaseType.BLOCK, RecipeType.BUCKET, FluidGUI::new, Fluid.class));
-		ModElementType.FOOD = register(
-				new ModElementType<>("food", 'f', BaseType.ITEM, RecipeType.ITEM, FoodGUI::new, Food.class));
+				new ModElementType<>("fluid", 'f', BaseType.BLOCK, RecipeType.BUCKET, FluidGUI::new, Fluid.class));
 		ModElementType.FUEL = register(
 				new ModElementType<>("fuel", null, BaseType.FUEL, RecipeType.NONE, FuelGUI::new, Fuel.class));
 		ModElementType.FUNCTION = register(
-				new ModElementType<>("function", null, BaseType.DATAPACK, RecipeType.NONE, FunctionGUI::new,
+				new ModElementType<>("function", 'u', BaseType.DATAPACK, RecipeType.NONE, FunctionGUI::new,
```

Comment:
u is already in use for fluid, did you mean f?

Reply:
Nope, I also changed the fluid shortcuts, so it uses `f`. I thought the fluid ME was used more often, so I changed its shortcuts for its first letter.
![image](https://user-images.githubusercontent.com/38427877/158221789-85679250-039a-4760-93c0-34b01e222c66.png)

Reply:
I see